packages/storage/docs/stories/UpdateUserPhotoModal.story.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { RainbowFirebaseApp } from '@rainbow-modules/app';
+import {
+    WhenAuthenticated,
+    WhenNoAuthenticated,
+    EmailPasswordSignInForm,
+} from '@rainbow-modules/auth';
+import { Avatar, Button } from 'react-rainbow-components';
+import { getAuth, onAuthStateChanged } from 'firebase/auth';
+import { Camera } from '@rainbow-modules/icons';
+import styled from 'styled-components';
+import app from '../../../../firebase';
+import UpdateUserPhotoModal from '../../src/components/UpdateUserPhotoModal';
+
+const StyledButton = styled(Button)`
+    border-radius: 50%;
+    width: 30px;
+    height: 30px;
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin: -25px 0 0 40px;
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 70px;
+    height: 70px;
+`;
+
+export const BasicUpdateUserPhotoModal = () => {
+    const [photoURL, setPhotoURL] = useState();
+    const [isOpen, setIsOpen] = useState(false);
+    const [initials, setInitials] = useState();
```

Comment:
Instead of using this states (photoURL and initials) you can use the useCurrentUser hook which will return null if unauthenticated or the user object when it is authenticated, then you can get the phototURL directly from the user object and the initials can be computed inline, thus avoiding using states.
Keeping track of states it is hard and lead to bugs, then always you need to think it twice to decide to use a state, in this case the isOpen state is needed but the others two do not.